ci-operator/jobs/Apicurio/apicurio-registry-system-tests/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- cspi-qe-ocp-lp
```

Comment:
Is there a reason that we left `rkubis` out of the approvers section here? I think it would be nice to allow more then just the cspi-qe team to approve.

Reply:
Okay, I will add @rkubis also as an approver here